sdk/identity/azure-identity/azure/identity/_credentials/azd_cli.py

```diff
@@ -91,7 +91,9 @@ def close(self) -> None:
         """Calling this method is unnecessary."""
 
     @log_get_token("AzureDeveloperCliCredential")
-    def get_token(self, *scopes: str, **kwargs: Any) -> AccessToken:
+    def get_token(
+        self, *scopes: str, claims: Optional[str] = None, tenant_id: Optional[str] = None, **kwargs
```

Comment:
Why we show it here while it is not supported?

Reply:
See https://github.com/Azure/azure-sdk-for-python/pull/31047#discussion_r1268704555

Reply:
We do need to improve the stand-in "not supported" docstring for parameters that aren't meant to be used -- we should be specific about exactly what behavior a user would see. But to match the protocol's signature, which is the only way to resolve type warnings, we have to include `claims` and `tenant_id` outside of `**kwargs`.